.changeset/two-hats-judge.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-techdocs': patch
+'@backstage/plugin-techdocs-backend': patch
+---
+
+added support for title in Techdocs
```

Comment:
```suggestion
Display [metadata.title](https://backstage.io/docs/features/software-catalog/descriptor-format#title-optional) for components on the TechDocs homepage, if defined; otherwise fall back to `metadata.name` as displayed before.
```

Thinking this could be a touch more descriptive 😄

Reply:
Thanks for the tweak, I'll create clearer descriptions next time. 😄